File: TestP1.rb
require_relative 'Controller/controller'
require_relative 'UI/textUI'
require_relative 'Game'
module Irrgarten

  
  NUM_JUGADORES = 3

  vista=UI::TextUI.new 
  juego=Game.new(NUM_JUGADORES)

  controller=Control::Controller.new(juego,vista)
  controller.play
end

File: java/src/irrgarten/TestP1.java
package irrgarten;

import irrgarten.controller.Controller;

import irrgarten.UI.GraphicUI;

public class TestP1 {
    public static void main(String[] args) {
        
         final int N_PLAYERS = 1;
        
         
        GraphicUI vista = new GraphicUI();
        Game juego = new Game(N_PLAYERS);
        Controller controlador = new Controller(juego, vista);

        controlador.play();
    }
}
